builtin/logical/pki/path_root.go

```diff
@@ -243,14 +247,23 @@ func (b *backend) pathCASignIntermediate(
 	case "pem":
 		resp.Data["certificate"] = cb.Certificate
 		resp.Data["issuing_ca"] = cb.IssuingCA
+		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
 
 	case "pem_bundle":
-		resp.Data["certificate"] = fmt.Sprintf("%s\n%s", cb.Certificate, cb.IssuingCA)
+		resp.Data["certificate"] = cb.ToPEMBundle()
 		resp.Data["issuing_ca"] = cb.IssuingCA
+		resp.Data["issuing_ca_chain"] = cb.IssuingCAChain
 
 	case "der":
 		resp.Data["certificate"] = base64.StdEncoding.EncodeToString(parsedBundle.CertificateBytes)
 		resp.Data["issuing_ca"] = base64.StdEncoding.EncodeToString(parsedBundle.IssuingCABytes)
+		for _, cert := range parsedBundle.IssuingCAChainBytes {
+			resp.Data["issuing_ca_chain"] = fmt.Sprintf("%s\n%s", resp.Data["issuing_ca_chain"], base64.StdEncoding.EncodeToString(cert))
```

Comment:
On first value this will have a leading newline. It seems odd. If you really can concatenate DER values, the better thing to use would be a `bytes.Buffer`...write values in and at the end pull the bytes out.